### PR TITLE
sync with english version for Quicklinks

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for Quicklinks

### CODE REVIEW

File: files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: 快速链接
+slug: MDN/Writing_guidelines/Page_structures/Quicklinks
+---
+
+{{MDNSidebar}}
+
+MDN 支持在页面中添加快速链接；这些框是包含到 MDN 上的其他页面或站外页面链接的潜在分层列表。本文描述了如何创建快速链接框。
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md:8]:
```suggestion
MDN 支持在页面中添加快速链接；这些框是包含到 MDN 上的其他页面或站外页面链接的可分层列表。本文描述了如何创建快速链接框。
```

File: files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+需要注意的重要事项：
+
+- **应该**使用有序列表。
+- 你可以通过在同一个 {{HTMLElement("li")}} 块中使用包含另一个有序列表的 {{HTMLElement("details")}} 来创建嵌套列表。
+
+## 使用宏创建快速链接
+
+值得注意的是，你可以（而且通常**应该**）使用宏来生成快速链接。每当你需要在多个页面上使用相同的快速链接集时，你都应该将它们转换为宏。
+
+你的宏可以尽可能简单或复杂；它需要输出类似于上面的[快速链接语法](#快速链接语法)中所示的 HTML。
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md:61]:
```suggestion
你的宏可以是简单或复杂的（若有必要）；它需要输出类似于上面的[快速链接语法](#快速链接语法)中所示的 HTML。
```

File: files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      </details>
+    </li>
+  </ol>
+</section>
+```
+
+需要注意的重要事项：
+
+- **应该**使用有序列表。
+- 你可以通过在同一个 {{HTMLElement("li")}} 块中使用包含另一个有序列表的 {{HTMLElement("details")}} 来创建嵌套列表。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/writing_guidelines/page_structures/quicklinks/index.md:55]: **inside**

```suggestion
- 你可以通过在同一个 {{HTMLElement("li")}} 块**内部**使用包含另一个有序列表的 {{HTMLElement("details")}} 来创建嵌套列表。
```

**Review by jasonren0403 — APPROVED** [review]
👍 Just one suggestion

**Pull request merged**